airflow/cli/cli_parser.py

```diff
@@ -626,6 +626,16 @@ def positive_int(value):
     ),
     action="store_true",
     default=False)
+ARG_CONN_EXPORT_FORMAT = Arg(
+    ('--format',),
+    help='Output format of the file for exporting the connections',
+    default='json',
+    type=str)
+ARG_CONN_EXPORT_FILE = Arg(
+    ('--output-file',),
+    help='Output file path for exporting the connections',
+    required=True,
+    type=str)
```

Comment:
Have you considered using argparse.Filetype?
https://docs.python.org/3/library/argparse.html#argparse.FileType
This way, you will be able to export data directly to stdout.
```
airflow connections export - | grep MY_CON
```